src/main/java/org/ilgcc/app/inputs/Gcc.java

```diff
@@ -31,7 +31,10 @@ public class Gcc extends FlowInputs {
     private String parentBirthMonth;
     private String parentBirthYear;
     private String parentBirthDate;
-    private String phoneNumber;
+  private String parentIsServing;
+  private String parentInMilitaryReserveOrNationalGuard;
+
+  private String phoneNumber;
```

Comment:
nit pick, align the spacing
```suggestion
    private String parentIsServing;
    private String parentInMilitaryReserveOrNationalGuard;

    private String phoneNumber;
```

Reply:
ope - ty